### ISSUE TITLE
Files not being parsed relative to challenge.yml file

### ISSUE THREAD

**Issue by jimender2** [CTFd/ctfcli#24]
I am still experiencing this error when running the version when I installed this using pip.

This is the pip and pip3 info.

```
$ pip3 show ctfcli
Name: ctfcli
Version: 0.0.4
Summary: Tool for creating and running Capture The Flag competitions
Home-page: UNKNOWN
Author: Kevin Chung
Author-email: kchung@ctfd.io
License: Apache 2.0
Location: /home/user/.local/lib/python3.6/site-packages
Requires: requests, colorama, appdirs, fire, cookiecutter, Pygments, click, pyyaml
Required-by: 
```

And the error I get is as follows.

```
$ ctf challenge sync Over_The_Top
Found Over_The_Top/challenge.yml
Loaded Over the Top
Syncing Over the Top
Traceback (most recent call last):
  File "/home/user/.local/bin/ctf", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.6/site-packages/ctfcli/__main__.py", line 74, in main
    fire.Fire(CTFCLI)
  File "/home/user/.local/lib/python3.6/site-packages/fire/core.py", line 138, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/home/user/.local/lib/python3.6/site-packages/fire/core.py", line 471, in _Fire
    target=component.__name__)
  File "/home/user/.local/lib/python3.6/site-packages/fire/core.py", line 675, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/home/user/.local/lib/python3.6/site-packages/ctfcli/cli/challenges.py", line 134, in sync
    sync_challenge(challenge=challenge)
  File "/home/user/.local/lib/python3.6/site-packages/ctfcli/utils/challenge.py", line 95, in sync_challenge
    files.append(("file", open(f, "rb")))
FileNotFoundError: [Errno 2] No such file or directory: 'src/main.c'
```

If I make a directory and file in the the folder I am running the file from, the command works just fine. But if I do not have it in that directory, it fails. It should be running it from path that is relative to the challenge.yml location.

Currently I am bypassing this (and recursivly searching folders) using this command.
```
find . -type d -exec echo '(cd {} && ctf challenge sync)' ';' | sh
```

**Comment by ColdHeat** [comment]
I actually believe this is resolved in the current master, it's just waiting for a release to pypi.

**Comment by ColdHeat** [comment]
I released `0.0.5` to PyPI which should have the fix for this.
